### PR TITLE
OSDOCS-7078: add ingress dash

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.15+

Issue: https://issues.redhat.com/browse/OSDOCS-7078

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@jotak @frobware could either of you PTAL at this docs PR for updating Network Observability docs to include Ingress dash? Thank you

**Comment by jldohmann** [comment]
@ShudiLi could you PTAL at this docs PR for QE ack? This adds Ingress dashboard information to Network Observability docs. thank you 🙇

**Comment by jotak** [comment]
cc @OlivierCazade

**Comment by OlivierCazade** [comment]
/lgtm
Thanks!

**Comment by ShudiLi** [comment]
/lgtm
Thanks

**Comment by skopacz1** [comment]
Preview link: [Ingress Operator dashboard](https://71012--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/networking-dashboards#ingress-dashboards)

File: networking/networking-dashboards.adoc [hunk 1]
```diff
@@ -21,3 +21,15 @@ You can view both general networking metrics as well as OVN-Kubernetes metrics f
 To view general networking metrics, select *Networking/Linux Subsystem Stats* from the *Dashboards* drop-down list. You can view the following networking metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
 
 To view OVN-Kubernetes metrics select *Networking/Infrastructure* from the *Dashboards* drop-down list. You can view the following OVN-Kuberenetes metrics: *Networking Configuration*, *TCP Latency Probes*, *Control Plane Resources*, and *Worker Resources*.
+
+[id="ingress-dashboards"]
+== Ingress Operator dashboard
+You can view networking metrics handled by the Ingress Operator from the dashboard. This includes things like the following:
```
Comment (skopacz1) [inline networking/networking-dashboards.adoc:27]: Suggestion: "things" reads just a bit informal and/or unspecific, maybe you can use a more specific word such as "metrics" even if it does repeat from the previous sentence:
```suggestion
You can view networking metrics handled by the Ingress Operator from the dashboard. This includes metrics like the following:
```

Up to you though!

File: networking/networking-dashboards.adoc [hunk 2]
```diff
@@ -21,3 +21,15 @@ You can view both general networking metrics as well as OVN-Kubernetes metrics f
...
+
+* Incoming and outgoing bandwidth
+* HTTP error rates
+* HTTP server response latency
+
+To view these Ingress metrics, select *Networking/Ingress* from the *Dashboards* drop-down list. You can view Ingress metrics for the following: *Top 10 Per Route*, *Top 10 Per Namespace*, and *Top 10 Per Shard*.
```
Comment (skopacz1) [inline networking/networking-dashboards.adoc:33]: Totally unimportant but maybe you could be more specific by describing what the "top 10 per" slices are called. Maybe "categories"?
```suggestion
To view these Ingress metrics, select *Networking/Ingress* from the *Dashboards* drop-down list. You can view Ingress metrics for the following categories: *Top 10 Per Route*, *Top 10 Per Namespace*, and *Top 10 Per Shard*.
```

Again, up to you

**Review by skopacz1 — COMMENTED** [review]
Left a few minor suggestions but otherwise looks great!

File: networking/networking-dashboards.adoc (hunk 1 above)
Comment (jldohmann) [reply networking/networking-dashboards.adoc:27]: good suggestion, thanks!

File: networking/networking-dashboards.adoc (hunk 2 above)
Comment (jldohmann) [reply networking/networking-dashboards.adoc:33]: yeah good point, they're kinda different from other dashboard entries...categories sounds good, thanks 😄

**Pull request merged**